containers/server-helm/templates/config.yaml

```diff
@@ -23,6 +23,7 @@ data:
   NO_SSL: "Y"
   MANAGER_MAIL_FROM: {{ .Values.uyuniMailFrom }}
   UYUNI_FQDN: {{ .Values.fqdn }}
+  TZ: {{ .Values.timeZone | default "Etc/UTC" }} 
```

Comment:
For documentation purposes you should add the new `timeZone` value in the `values.yaml`